caffe2/operators/reverse_packed_segs_op.cu

```diff
@@ -2,6 +2,91 @@
 #include "reverse_packed_segs_op.h"
 
 namespace caffe2 {
+
+namespace {
+
+template <typename T, typename LengthType>
+__global__
+void ReversePackedSegments_kernel(
+      size_t max_length,
+      size_t batch_size,
+      size_t block_size,
+      const LengthType* lengths_ptr,
+      const T* data_ptr,
+      T* rev_data_ptr) {
+  
```

Comment:
Mega nit, trailing whitespace. Internal linter trips over this. Could you configure your editor to automatically strip this?